### ISSUE TITLE
Tokens logged by the Custos backend may have security implications

### ISSUE THREAD

**Issue by VJalili** [galaxyproject/galaxy#10167]
The custos authnz backend logs tokens it receives from IdPs: 

https://github.com/galaxyproject/galaxy/blob/2dc39a5c0158387396ec8b2018d29826984780d2/lib/galaxy/authnz/custos_authnz.py#L68

Hence dumping information such as the following to the Galaxy log: 

```shell
galaxy.authnz.custos_authnz DEBUG 2020-08-31 00:20:48,416 [p:1987,w:1,m:0] [uWSGIWorker1Core0] token={
 "access_token": "...",
 "expires_in": 60,
 "refresh_expires_in": 1800,
 "refresh_token": "...",
 "token_type": "bearer",
 "id_token": "...",
 "not-before-policy": 0,
 "session_state": "...",
 "scope": [
  "openid",
  "email",
  "profile"
 ],
 "expires_at": 1598858508.416065
}
```

These information, the tokens in particular, should be securely stored as they can be exploited to impersonate the user. The database where these tokens are stored can be encrypted; however not sure how securely the logs are stored.

**Comment by dannon** [comment]
Thanks for noticing this; looks like it was likely debugging cruft left in.  Fixed in #10168

**Comment by VJalili** [comment]
Thanks @dannon
